test-utils/runtime-tester/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod run_test;
```

Comment:
Let's add a short doc comment explaining what this crate is about. 

Right now, if an new person comes to the codbases and sees the code, it'd be difficult for them to understand what's this is for.